File: src/main/kotlin/limia/Definition/GlobalConstants.java
package limia.Definition;

/**
 * Created by macbook on 9/4/17.
 */
public class GlobalConstants {
    public static class HTTPMessages {
        public static final String CREATE = "created";
        public static final String READ = "read";
        public static final String UPDATE = "updated";
        public static final String DELETE = "deleted";
        public static final String ALREADY_EXISTS = "already exists";
    }

    public static enum CRUD {CREATE, READ, UPDATE, DELETE}
}

File: src/main/kotlin/limia/Definition/GlobalConstants.kt
package limia.Definition

/**
 * Created by macbook on 9/4/17.
 */
open class GlobalConstants {
    object HTTPMessages {
        val CREATE = "created"
        val READ = "read"
        val UPDATE = "updated"
        val DELETE = "deleted"
        val NOT_FOUND = "not found"
        val EXISTS = "exists"
        val ALREADY_EXISTS = "already exists"
    }

    enum class CRUD {
        CREATE, READ, UPDATE, DELETE
    }
}
